### ISSUE TITLE
cant dismount flying enemy player

### ISSUE THREAD

**Issue by kolgarlol** [elysium-project/tbc-bug-tracker#1085]
**Summary:**  Dismounting enemy players dont works

**Steps to Reproduce:** I had tried to stun/repeatance/judgement of justice/dps flying players, but cant dismount them. 

**Expected Results:** dismount flying enemy players

**Actual Result:** immune to stuns, repeatance while enemy players in flight form/at fly mount, and cant dismount them while damaging (had tried to autoattack flying player, crusader strike them, damage them with judgements, seals)

**Additional Information:**
I had tried to find some guides to dismount flying players, but cant find any information about it. But, to kill enemies in world pvp, I should know that they cant run away from me. I found 2 very old videos from lich king addon, where blast wave and taphoon dismounts flying player. I had found some messages from old forums, where players says that dragon breath (disorient effect) 100% dismounts players, thunderstorm of shaman 100% dismounts flying player...Deadly Throw of rogue, Shattering Throw of warrior, physical dot/bleed damage, ranged attacks, but CANT find any information about any other classes, especially about dismounting enemies by paladins. I have 2 videos about it here 1) https://www.youtube.com/watch?v=-fzMy_V_tS4 2) https://www.youtube.com/watch?v=pWlBr_fT6bI&t=160s Really, dont wanna play game, where player can just escape pvp with fly mount.

**Comment by Senjiu** [comment]
https://wowwiki.fandom.com/wiki/Knock_back

"Before patch 3.3 WotLK, receiving a knock back could also cause a player to become dismounted from their ground or flying mount."

Both your videos are from WotLK. In TBC there are no knockback effects. In fact, according to https://wow.gamepedia.com/Blast_Wave the knockback effect to blastwave was added in Patch 3.0. Your second video, although it happened in TBC already had the WotLK talents, it was after the WotLK pre-patch, shortly before the release of the expansion. Otherwise the druid would not have had the Typhoon talent.

In summary, the ability to dismount players from flying mounts like that only existed for a short period between patch 3.0 and 3.3 (well, it was probably half a year or so, not too short but considering the game is 15 years old it's not so much).
If you want to kill a player on his flying mount you have to apply enough damage for him to die mounted.
